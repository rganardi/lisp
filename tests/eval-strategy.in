(define a (quote yes))
(define b (quote no))

((lambda (a b) b) b a)
