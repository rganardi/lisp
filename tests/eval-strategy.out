parse_eval: unspecified
parse_eval: unspecified
yes
